File: Quiz/Functions & Structs/Problem1.cpp
#include<iostream>
using namespace std;

int PyramidVolume(int l, int w, int h);

int main() {

	cout << PyramidVolume(3,4,5);

}

int PyramidVolume(int l, int w, int h) {

	return (l * w * h) / 3;

}

File: Quiz/C++ Review/Problem1.py
#include<iostream>
using namespace std;

int main() {
	int num;
	cout << "Enter an integar: ";
	cin >> num;
	if (num % 5 == 0)
		cout << num << " is a multiple of 5";
	else
		cout << num << " is not a multiple of 5";


}
